app/views/research_sessions/preview.html.erb

```diff
@@ -114,6 +114,23 @@
       collected as part of the research is stored securely and kept strictly
       confidential.</p>
   </section>
+  <section>
+    <h3 class="subtitle-small" id="private">Concerns and complaints</h3>
+
+    <p>If you have any concerns or a complaint, or would like to have your data deleted please contact:
+
+    <dl class="definition-list">
+      <div>
+        <dt class="definition-list__term">Telephone:</dt>
+        <dd>02084987869</dd>
```

Comment:
Would it be worth making this a link with `href="tel:+442084987869"` to help folk browsing from outside the UK correctly contact the correct number.

Reply:
As mentioned offline, this page is for printing so the functionality to call isn't needed here. However prepending the number with +44 should be done.